docs/user-guide/energy.md

```diff
@@ -0,0 +1,126 @@
+# Energy use
+
+This section covers how to monitor energy use for your jobs on ARCHER2 and how to control the CPU frequency
+which allows some control over how much energy is consumed by jobs.
+
+!!! important
+    The default CPU frequency on ARCHER2 compute nodes is currently unset. This means that the cores will
+    run at the maximum frequency available - 2.25 GHz. We anticipate that the default CPU frequency will
+    be set to 2.0 GHz in early December 2022 to improve the energy efficiency of the ARCHER2 system.
+
+## Monitoring energy use
+
+The Slurm accounting database stores the total energy consumed by a job and you can also directly access
+the counters on compute nodes which capture instantaneous power and energy data broken down by different
+hardware components.
+
+### Using sacct to get energy usage for individual jobs
+
+Energy usage for a particular job may be obtained using the `sacct` command. For instance
+
+```
+sacct -j 2658300 --format=JobID,Elapsed,ReqCPUFreq,ConsumedEnergy
+```
+
+will provide the elapsed time and consumed energy in joules for the job(s) specified with `-j`.
+The output of this command is: 
+
+```
+JobID           Elapsed ReqCPUFreq ConsumedEnergy 
+------------ ---------- ---------- -------------- 
+2658300        02:19:48    Unknown          4.58M 
+2658300.bat+   02:19:48          0          4.58M 
+2658300.ext+   02:19:48          0          4.58M 
+2658300.0      02:19:09    Unknown          4.57M 
+```
```

Comment:
I don't know if adding `-X` makes things easier or more complicated. If we execute `sacct [...] -X`, the output is
```
:~> sacct -j 2658300 --format=JobID,Elapsed,ReqCPUFreq,ConsumedEnergy -X
JobID           Elapsed ReqCPUFreq ConsumedEnergy
------------ ---------- ---------- --------------
2658300        02:19:48    Unknown          4.58M
```
Further info about `-X` option:
```
-X, --allocations
          Only show statistics relevant to the job allocation itself, not taking steps into consideration.
          NOTE: Without including steps, utilization statistics for job allocation(s) will be reported as zero.
```
What do you think?

Reply:
I think I prefer the full output with multiple job steps rather than summarising the whole job. Many users have multiple job steps in their job script and the more complete output with individual energy for each of the actual application runs.